pkg/sypgp/sypgp.go

```
@@ -461,6 +514,10 @@ func GenKeyPair(keyServiceURI string, authToken string) (entity *openpgp.Entity,
 
 // DecryptKey decrypts a private key provided a pass phrase
 func DecryptKey(k *openpgp.Entity) error {
+	if k == nil {
+		return nil
```

should return an error because this is a misuse of this function